components/AnimatedImage/library/src/commonMain/kotlin/org/jetbrains/compose/animatedimage/AnimatedImage.kt

```diff
@@ -2,13 +2,30 @@ package org.jetbrains.compose.animatedimage
 
 import androidx.compose.ui.graphics.ImageBitmap
 
+/**
+ * Represents an image formed by a sequence of frames with a specific duration each one.
+ */
 expect class AnimatedImage
 
+/**
+ * Loads an [AnimatedImage] from a given path.
```

Comment:
I thought about it but I wasn't completly sure, as the docs it's in the commonMain, so the sentence wouldn't be true if the method was used in compose for mobile/web. Should I still update it? Or can the docs be changed just for the desktop module?

Reply:
> Oh, you are right. But let's write that it is platform specific, and add info about each supported platform (we support only desktop at the moment)

Shouldn't we add it in the readme file rather than in the methods docs? Talking about the README, should I add one with a summary about the component?

Reply:
It is rare when people read Readme :). If we can provide information closer to code, we should do that.

Reply:
Got it! I'll push it in a moment then